airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/record_buffer/SerializableBuffer.java

```diff
@@ -37,6 +37,8 @@ public interface SerializableBuffer extends AutoCloseable {
    */
   long accept(AirbyteRecordMessage record) throws Exception;
 
+  long accept(String recordString, long emittedAt) throws Exception;
```

Comment:
add java doc. Correct me if I'm wrong, I think we want to note that we should move to a world where we pass the serialised data around, and not the entire airbyte record.

Reply:
Good call-out